packages/client/hmi-client/src/components/models/tera-typed-model-diagram.vue

```diff
@@ -364,8 +386,10 @@ watch(
 watch(
 	numberTypedNodes,
 	() => {
-		if (numberTypedNodes.value === numberNodes.value) {
+		if (allNodesTyped.value) {
 			emit('all-nodes-typed', typedModel.value);
+		} else {
+			emit('not-all-nodes-typed', numberNodes.value - numberTypedNodes.value);
```

Comment:
Event emitted so that `tera-stratify` can keep track of whether all nodes have have been typed. Just `'all-nodes-typed'` was not sufficient because if the user fully types the model, but then unchecks some of their selections, `tera-stratify` would not know about it.